resources/file.rb

```diff
@@ -5,12 +5,27 @@
 # Author:: Jamie Winsor (<jamie@vialstudios.com>)
 #
 
-actions :extract
-default_action :extract
+property :path, kind_of: String, name_property: true, required: true
+property :owner, kind_of: String
+property :group, kind_of: String
+property :mode, kind_of: Integer, default: 0755
```

Comment:
mode should probably accept a string as well as a integer and default to '0755' since we're pushing people to use strings for modes